certs/eventstore.conf

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
eventstore has a nice wizard to create these files these days

this was autogenerated by that